recipe/meta.yaml

```diff
@@ -13,6 +13,7 @@ source:
     patches:
       - dont_link_with_libcxxabi.patch
       - 530-ld64-add-conda-specific-env-vars-to-modify-lib-search-paths.patch
+      - fix_sparc_pc.patch
```

Comment:
Can you send this patch to https://github.com/tpoechtrager/cctools-port ?

Reply:
Sure, I will check, where I can place that patch in that repro. It might be later today

Reply:
Done. See https://github.com/tpoechtrager/cctools-port/pull/81